kit/Kit.cpp

```diff
@@ -606,6 +597,7 @@ class Document final : public DocumentManagerInterface
 #endif
         }
 
+        // TODO: Should I remove the comment?
```

Comment:
Should I remove the comment?

Reply:
This question still needs an answer.

Reply:
Oh - yes please do =) there is no deadlock there anymore.